clients/drcachesim/tests/reuse_distance.templatex

```diff
@@ -1,8 +1,10 @@
 Hello, world!
 ---- <application exited with code 0> ----
 Reuse distance tool aggregated results:
-Total accesses: [0-9]+
-Unique accesses: [0-9]+
+Total accesses: [1-9][0-9]+
+Instruction accesses: [1-9][0-9]+
+Data accesses: [1-9][0-9]+
+Unique accesses: [1-9][0-9]+
```

Comment:
That statistic (and most others as well) is only available for the combined access stream.  So while it might be interesting to separate this, that would require a much more intrusive and potentially performance impacting change.  The split done in this PR is only for the distance histogram, with no changes or duplication to the expensive cacheline tracking logic.